tag-security-and-privacy.md

```diff
@@ -0,0 +1,75 @@
+# TAG Review: Security and Privacy questionnaire
+
+## 2.1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+There is an ongoing discussion ([tc39/proposal-import-attributes#61](https://github.com/tc39/proposal-import-attributes/issues/61)) about communicating the `type` attribute to the origin. For instance:
+
+```js
+import json from "/foo.json" with type: "json";
+```
+
+would presumably send `Accept: application/json` to the origin.
+
+## 2.2. Is this specification exposing the minimum amount of information necessary to power the feature?
+
+Yes.
+
+## 2.3. How does this specification deal with personal information or personally-identifiable information or information derived thereof?
+
+No extra personal information or personally-identifiable information is exposed by this proposal.
+
+## 2.4. How does this specification deal with sensitive information
+
+No extra sensitive information is exposed by this proposal.
+
+## 2.5. Does this specification introduce new state for an origin that persists across browsing sessions?
+
+No.
+
+## 2.6. What information from the underlying platform, e.g. configuration data, is exposed by this specification to an origin?
+
+Regardless of the conclusion of 2.1 about exposing the type to the origin; if a host implements this proposal, JSON module must be supported.
+
+## 2.7. Does this specification allow an origin access to sensors on a user’s device
+
+No.
+
+## 2.8. What data does this specification expose to an origin? Please also document what data is identical to data exposed by other features, in the same or different contexts.
+
+No data.
+
+## 2.9. Does this specification enable new script execution/loading mechanisms?
+
+This proposal will enable future module types on the web, first JSON module and then HTML, CSS, ... The `type` attribute is designed to prevent execution from certain module types.
+
+## 2.10. Does this specification allow an origin to access other devices?
+
+No.
+
+## 2.11. Does this specification allow an origin some measure of control over a user agent’s native UI?
+
+No.
+
+## 2.12. What temporary identifiers might this this specification create or expose to the web?
+
+No identifiers.
+
+## 2.13. How does this specification distinguish between behavior in first-party and third-party contexts?
+
+No difference.
+
+## 2.14. How does this specification work in the context of a user agent’s Private Browsing or "incognito" mode?
+
+No difference.
+
+## 2.15. Does this specification have a "Security Considerations" and "Privacy Considerations" section?
+
+Part of the proposal motivation is the security aspect, available in the README. It doesn't provide privacy considerations yet.
```

Comment:
s/It doesn't provide privacy considerations/We consider that there are no particular privacy considerations./ (unless you think we should actually add this)

Reply:
I agree we should remove it, thanks